src/main/webapp/WEB-INF/views/UserPageVer1.2.html

```diff
@@ -0,0 +1,373 @@
+                                            <div class="col-xs-6">
+                                                <div>min</div>
+                                                <div><i class="fa fa-dollar"></i>100000</div>
+                                            </div>
+                                            <div class="col-xs-6">
+                                                <div class="right-box">Still needed</div>
+                                                <div class="right-box"><i class="fa fa-dollar"></i>9999</div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="card">
+                                    <a href="#"><img class="card-img-top img-responsive"
+                                                     src="https://goo.gl/e7PZpL"></a>
+                                    <div class="card-block">
+                                        <a href="#"><h4 class="card-title">Artificial Intelligence A-Z™: Learn How To
+                                            Build An AI</h4></a>
+                                        <div class="card-text">
+                                            <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                                                tempor
+                                                incididunt ut labore et
+                                                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                                                ullamco
+                                                laboris nisi ut aliquip
+                                                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                                                voluptate
+                                                velit
+                                                esse cillum dolore eu
+                                                fugiat nulla pariatur.
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="card-footer">
+                                        <div class="row">
+                                            <div class="col-xs-6">
+                                                <div>min</div>
+                                                <div><i class="fa fa-dollar"></i>100000</div>
+                                            </div>
+                                            <div class="col-xs-6">
+                                                <div class="right-box">Still needed</div>
+                                                <div class="right-box"><i class="fa fa-dollar"></i>9999</div>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)